src/test/java/com/puppycrawl/tools/checkstyle/JavadocPropertiesGeneratorTest.java

```diff
@@ -281,6 +365,15 @@ public void testNotConstants(@SysErr Capturable systemErr, @SysOut Capturable sy
             .isEqualTo(0);
     }
 
+    /**
+     * Tests JavadocPropertiesGenerator.
+     *
+     * @param systemErr wrapper for {@code System.err}
+     * @param systemOut wrapper for {@code System.out}
+     * @noinspection JUnitMalformedDeclaration
+     * @noinspectionreason JUnitMalformedDeclaration -
+     *      we must test CLI using injected system out/err
```

Comment:
too much false positives, lets put this annotation on whole Class for this class and MainTest.